cmd/check/main.go

```diff
@@ -26,6 +28,8 @@ func main() {
 		ForceColors: true,
 	})
 
+	logs.Progress = log.New(os.Stderr, "go-cr-progress-", log.LstdFlags)
```

Comment:
Do we not need to set `logs.Warn` here?

Also how will that prefix make the logs look? The prefix here seems like `lager` style but this is the `log` package so I'm not sure how it'll look.

Reply:
Whoops, fixed.

Also, dropped the prefixes for now, to make it more similar to `go-retryablehttp` and `logrus`. 
Overall, there the 3 sources still have differing log formats :(

Here is a sample 
```
INFO[0000] pushing sha256:c7a1f27b34628052fac814ae20606e7c8b28648f912d4e33269005713b214a23 to index.docker.io/xtremesameer/test:latest
2019/11/21 00:25:58 [DEBUG] GET https://index.docker.io/v2/
2019/11/21 00:25:59 [DEBUG] GET https://auth.docker.io/token?scope=repository%3Axtremesameer%2Ftest%3Apush%2Cpull&service=registry.docker.io
.
.
.
2019/11/21 00:26:00 existing blob: sha256:1eb51d2d74493e2ce1616b8d267a92de850ae19e438e1dc6a92b1f18fef6a9fd
2019/11/21 00:26:00 [DEBUG] PUT https://index.docker.io/v2/xtremesameer/test/blobs/uploads/f70d8e70-925a-4388-8fba-ca06f5f8f7c0?_state=I4FUwcugPU3qAFFIoOjUN7iu_H7Xz21NtOOj3x84Gz97Ik5hbWUiOiJ4dHJlbWVzYW1lZXIvdGVzdCIsIlVVSUQiOiJmNzBkOGU3MC05MjVhLTQzODgtOGZiYS1jYTA2ZjVmOGY3YzAiLCJPZmZzZXQiOjE2OTcsIlN0YXJ0ZWRBdCI6IjIwMTktMTEtMjFUMDU6MjY6MDBaIn0%3D&digest=sha256%3A5cd04195ca614ade8b14798b2391d1e60019262d8127eb2115621fba7b58efc7
2019/11/21 00:26:01 pushed blob: sha256:5cd04195ca614ade8b14798b2391d1e60019262d8127eb2115621fba7b58efc7
2019/11/21 00:27:41 xtremesameer/test:latest: digest: sha256:c7a1f27b34628052fac814ae20606e7c8b28648f912d4e33269005713b214a23 size: 594
INFO[0103] pushed
```